pkg/redo/config.go

```diff
@@ -155,13 +162,21 @@ func IsBlackholeStorage(scheme string) bool {
 
 // InitExternalStorage init an external storage.
 var InitExternalStorage = func(ctx context.Context, uri url.URL) (storage.ExternalStorage, error) {
+	s, err := util.GetExternalStorageWithTimeout(ctx, uri.String(), DefaultTimeout)
+	if err != nil {
+		return nil, errors.WrapChangefeedUnretryableErr(errors.ErrStorageInitialize, err)
+	}
+	return s, nil
+}
+
+func initExternalStorageForTest(ctx context.Context, uri url.URL) (storage.ExternalStorage, error) {
```

Comment:
There is no test file which uses `initExternalStorageForTest`, maybe `initExternalStorage` is enough?

Reply:
[ValidateStorage](https://github.com/pingcap/tiflow/pull/8181/files#diff-c39f6b418c48bd2b7281aa46aa529d775dffc376505422d2e4ebff54630da0b7R187) use this method to test external storage when create changefeed.